statsmodels/stats/contingency_tables.py

```diff
@@ -1402,3 +1492,1494 @@ def cochrans_q(x, return_object=True):
         return b
 
     return q_stat, pvalue, df
+
+
+def _shift_zeros(dataframe):
+    """
+    If table contains a cell with zero observations, jitter it up by 0.5
+
+    Chi-squared tests can't handle having expected counts of zero.
+
+    Parameters
+    ----------
+    dataframe : pd.DataFrame
+        dataframe of data to shift
+
+    Return
+    ------
+    pd.DataFrame
+        shifted data
+    """
+    if dataframe.min().min() == 0:
+        return dataframe + 0.5
```

Comment:
Do you want to increase all values, or should this just increase the zeros to 0.5?